pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -111,7 +111,7 @@ func (pc *PackageControllerClient) EnableCuratedPackages(ctx context.Context) er
 		chartName = chartName + "-" + pc.clusterName
 	}
 
-	if err := pc.chartInstaller.InstallChart(ctx, chartName, ociURI, pc.chart.Tag(), pc.kubeConfig, "", valueFilePath, values); err != nil {
+	if err := pc.chartInstaller.InstallChart(ctx, chartName, ociURI, pc.chart.Tag(), pc.kubeConfig, "eksa-packages", valueFilePath, values); err != nil {
```

Comment:
helm is going to create that namespace (https://github.com/aws/eks-anywhere/blob/main/pkg/executables/helm.go#L133), but meanwhile the chart itself has a resource for that namespace as well. I'm wondering if the chart installation will have an issue.

Reply:
This is addressed in https://github.com/aws/eks-anywhere-packages/pull/842